### PR TITLE
rewriting checks for Wanda gatherer corosync-cmapctl

### PR DESCRIPTION
**Pull request by pirat013**
the following checks are done:
00081D.yaml  15F7A8.yaml  53D035.yaml  7E0221.yaml  822E47.yaml  D78671.yaml  FB0E0D.yaml

### CODE REVIEW

**Review by nelsonkopliku — APPROVED** [review]
:rocket:

File: priv/catalog/00081D.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-example-for-etccorosynccorosync-conf
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-example-for-etccorosynccorosync-conf
+
+facts:
+  - name: facts_max_messages
```
Comment (arbulu89) [inline priv/catalog/00081D.yaml:46]: We don't need this `facts` prefix here. We already have the namespace

**Review by arbulu89 — COMMENTED** [review]
@pirat013 
The checks looks great!
The unique thing, I would remove that `facts_` prefix from the facts name. They already have the `facts.` namespace so it is some sort of redundant

File: priv/catalog/00081D.yaml (hunk 1 above)
Comment (pirat013) [reply priv/catalog/00081D.yaml]: Thanks a lot. A new commit is available. 34f4c0131e53bacb1ec43f91d629d91c88809a9a

**Review by arbulu89 — APPROVED** [review]
Thanks @pirat013

**Review by arbulu89 — CHANGES_REQUESTED** [review]
I think something went wrong with the `156F64` modification

**Comment by pirat013** [comment]
The file 156F64.yaml is not part of the corosync_smapctl gatherer. The file is maintained already in the PR #115 and therefor I've remove the file from this PR.

**Comment by nelsonkopliku** [comment]
Just a heads up: `corosync-cmapctl` has been recently updated here https://github.com/trento-project/agent/pull/168, enabling support for check `32CFC6 Corosync is running with at least 2 rings` have been merged here.

Up to you if you prefer adding it in this PR or in another one.

**Review by arbulu89 — APPROVED** [review]
Thanks!

**Pull request merged**